actions/treasures.go

```diff
@@ -23,65 +21,21 @@ type treasureReq struct {
 }
 
 type treasureRes struct {
-	Success string `json:"success"`
+	Success bool `json:"success"`
 }
 
+var IotaWrapper = services.IotaWrapper
+
 // Verifies the treasure and claims such treasure.
 func (t *TreasuresResource) VerifyAndClaim(c buffalo.Context) error {
 	req := treasureReq{}
 	oyster_utils.ParseReqBody(c.Request(), &req)
 
 	addr := models.ComputeSectorDataMapAddress(req.GenesisHash, req.SectorIdx, req.NumChunks)
-	iotaAddr := make([]giota.Address, len(addr))
-
-	for i, address := range addr {
-		iotaAddr[i] = giota.Address(address)
-	}
-
-	transactionsMap := services.FindTransactions(iotaAddr)
-
-	if len(transactionsMap) != len(iotaAddr) {
-		// indicate that PoW failure.
-	}
-
-	passedTimestamp := time.Now().AddDate(-1, 0, 0)
-	for _, iotaAddress := range iotaAddr {
-		if _, hasKey := transactionsMap[iotaAddress]; !hasKey {
-			// indicate that PoW failure
-		}
-
-		transactions := transactionsMap[iotaAddress]
-		// Check one the transactions has submit within the passed 1 year.
-		isTransactionWithinTimePeriod := false
-		for _, transaction := range transactions {
-			if transaction.Timestamp.After(passedTimestamp) {
-				isTransactionWithinTimePeriod = true
-				break
-			}
-		}
-		if !isTransactionWithinTimePeriod {
-			// Indicate that PoW failure
-			break
-		}
-	}
-
-	//ftr := &giota.FindTransactionsRequest{Bundles: []giota.Trytes{"DEXRPLKGBROUQMKCLMRPG9HFKCACDZ9AB9HOJQWERTYWERJNOYLW9PKLOGDUPC9DLGSUH9UHSKJOASJRU"}}
-	//resp, err := api.FindTransactions(ftr)
-	//
-	//datamap1, vErr := models.GetDataMap(req.GenesisHash, req.NumChunks)
-	//for _, d := range datamap1 {
-	//
-	//}
-	// or
-
-	//vErr, error := models.BuildDataMaps(req.GenesisHash, req.NumChunks)
-	//datamap, err := models.GetDataMapByGenesisHashAndChunkIdx(req.GenesisHash, req.NumChunks)
-	//for _, d := range datamap {
-	//
-	//}
+	verify, err = IotaWrapper.VerifyTreasure(addr)
```

Comment:
Instead of:
verify, err = IotaWrapper.VerifyTreasure(addr)
Try:
verify, err := IotaWrapper.VerifyTreasure(addr)    

Also, what IDE are you using?  Most IDEs have some kind of intellisense you can set up where it will tell you these errors.